README.md

```diff
@@ -120,13 +120,15 @@ from bleanser.core.modules.json import JsonNormaliser, delkeys, Json
 
 
 class Normaliser(JsonNormaliser):
-    # here, j is a dict, each file that this gets passed from the CLI call below
-    # is pre-processed by the cleanup function
+    # here, j is a dict, each file that this gets passed from the CLI call
+    # below is pre-processed by the cleanup function
     def cleanup(self, j: Json) -> None:
```

Comment:
I guess return type should be `Json` as well?

Reply:
oh, yeah, it should